### ISSUE TITLE
NieR (Replicant) (BLJM60223 & BLUS30481) Hang when saving with native UI.

### ISSUE THREAD

**Issue by Zero-Tang** [RPCS3/rpcs3#9770]
## Quick summary

I tried both versions. There is already a save data (saved right after the prologue). However, when I am going to save for next data, like to immediately save again or to play for some while, the save UI freezes within one or two seconds and I thereby have to quit, with my progress lost.

#### 1. Please attach RPCS3's log.
[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/5972547/RPCS3.log.gz)

#### 3. Please attach screenshots of your problem.
![Capture](https://user-images.githubusercontent.com/37060901/107790499-c94cc500-6d8d-11eb-8773-fc669ecdc4a6.JPG)

#### 4. Please provide comparison with real PS3.
Real PS3 does not hang whenever I try to save.

#### 5. Please provide your system configuration:
* OS Windows 10 LTSC 2019
* CPU AMD Ryzen Threadripper 2950X
* GPU Leadtek Nvidia GeForce 1080Ti
* Driver version v461.40
* etc. PS4 Controller over DS4Windows

#### Please include.
* Anything else you deem to be important
There is one line of log that looks suspicious. It appears right after the game hangs and it is also the last line of log before I quit the game.
```log
·F 0:04:18.984398 {PPU[0x1000000] Thread (main_thread) [0x00f6684c]} VM: Access violation reading location 0x41ca71ac (unmapped memory) [type=u64]
```

**Comment by Augusto7743** [comment]
That issue also happen in others games.
Had an game that only start if disabling Use Native User Interface because allways crash in "game saving screen".
Try in configuration emulator disabling Use Native User Interface and see if is "fixed".

**Comment by Zero-Tang** [comment]
This surely fixed the issue. The only problem is that the saving and loading becomes quite inconvenient. Anyway, little does better than nothing. It is great to see I could save my progress. I'd really appreciate that.
I would leave this issue open to see if RPCS3 team would like to fix "Native User Interface" or not.

**Comment by Augusto7743** [comment]
Have others games crashing if enabling Native User Interface.
Have game that in RPCS3 previous versions crash and current version not more crash, but now other games are crashing.

**Comment by Cursetin** [comment]
> That issue also happen in others games.
> Had an game that only start if disabling Use Native User Interface because allways crash in "game saving screen".
> Try in configuration emulator disabling Use Native User Interface and see if is "fixed".

Thank you, this fixed it for me as well, could not even load my old save files from last year

**Comment by Megamouse** [comment]
can you please disable all patches and see if it works?
Just to make sure it's not the issue.

**Comment by Zero-Tang** [comment]
I tried on latest version (0.0.15-11844-b5d4b5a2). It turns out that the emulator won't freeze if patches are disabled.

**Comment by Megamouse** [comment]
Patches do not concern us. you're using them at your own risk.
I'll close this issue because you said it works without the patches.

Feel free to reopen this issue if you encounter the problem without any patches applied.

**Comment by Teashrock** [comment]
@Megamouse 
Hello, running clean vanilla RPCS3 from nightly update. Ran into the same issue. Workaround is to disable native user interface, but it's still a bug, I have it. Please reopen.

**Comment by b0ugainvillea** [comment]
> @Megamouse
> Hello, running clean vanilla RPCS3 from nightly update. Ran into the same issue. Workaround is to disable native user interface, but it's still a bug, I have it. Please reopen.
> 
> Win 10 x64
> RPCS3-0.0.15-12120-7618e7f3 Alpha
> BLJM60223

Are you using the patch (Unlock FPS)? The patch is what is bugging the save when you do use it with Native User Interface (NUI) activated. Just like what was discussed prior, the patches isn't really a part of the emulator so they can't really do a thing about it because the emulator works just fine. I tried to play today and saw that still freezing when I was using the patch and try to save with NUI activated, but with it disabled I could save normally. On counterpart I could use NUI without any problems after I disabled the patch. They just doesn't work well together, choose one I go with it.
